usb_protocol/types/descriptors/uac3.py

```diff
@@ -71,6 +63,14 @@ class AudioInterfaceProtocolCodes(IntEnum):
     IP_VERSION_03_00 = 0x30
 
 
+class AudioFunctionProtocolCodes(IntEnum):
+    # As defined in [Audio30], Table A-3
+    FUNCTION_PROTOCOL_UNDEFINED = 0x00
+    AF_VERSION_01_00 = AudioInterfaceProtocolCodes.IP_VERSION_01_00
+    AF_VERSION_02_00 = AudioInterfaceProtocolCodes.IP_VERSION_02_00
+    AF_VERSION_03_00 = AudioInterfaceProtocolCodes.IP_VERSION_03_00
+
+
```

Comment:
Note this needed to move because it is making use of values from one of the other enums (needs to come after AudioInterfaceProtocolCodes)

Reply:
This one makes sense, but I still have questions about a couple of fix commits (mainly what makes sens for internal imports).  I'm redoing this PR against a different (local) branch.  Thanks.